python/instrumentation/openinference-instrumentation-dspy/pyproject.toml

```diff
@@ -37,6 +37,7 @@ instruments = [
 test = [
   "dspy-ai==2.1.0",
   "opentelemetry-sdk",
+  "pytest",
```

Comment:
why do we need this?

Reply:
I didn't see this was already defined in `dev-requirements.txt`, will remove.